apps/website/src/examples/Avatar.picture.tsx

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Avatar, getUserColor } from '@itwin/itwinui-react';
+
+export default () => {
+  return (
+    <Avatar
+      abbreviation='GB'
+      backgroundColor={getUserColor('Greg Bentley')}
+      image={<img src='https://www.bentley.com/wp-content/uploads/greg-bentley-hr-profile.jpeg' />}
```

Comment:
Where can I host an image to use instead of this one from bentley.com?  I don't want to use https://itwin.github.io/iTwinUI/css/assets/user-placeholder.png because it looks too close to the SVG examples.

Reply:
you can put it in the `public` folder under `apps/website` and it should be available at this url: `https://itwinui.bentley.com/<your-file-name.jpg>` (after it's deployed)

if you decided to do this, i would recommend _not_ using a human face (even if it's AI-generated). Instead, you could use something like this: https://stock.adobe.com/search?k=robot+avatar&asset_id=359684982
![](https://user-images.githubusercontent.com/9084735/217340305-6eef0874-b209-4354-b9e8-781ef6d06901.png)

Reply:
oh i just realized this is for the demos. we probably don't want to show the full domain in the code, so what you really need is to put an image inside the sandbox.

there might be cleaner ways of doing this, but one easy option is to convert it to a [data uri](https://css-tricks.com/data-uris/) and use it in the `src` attribute of the img.
```
<img src='data:image/...'>
```

the url is not the focus, so it gets the point across (that you can use an <img> tag)

Reply:
Good suggestion, replaced with that robot & used a data URI.

Reply:
oh wow that url is incredibly long. maybe move it out to a variable
```
const imgSrc = `data:image/...`;
```
```
<img src={imgSrc} />
```

Reply:
> move it out to a variable

bump

Reply:
looks good